compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -217,7 +213,7 @@ object desugar {
     val meth @ DefDef(_, tparams, vparamss, tpt, rhs) = transformQuotedPatternName(meth0)
     val methName = normalizeName(meth, tpt).asTermName
     val mods = meth.mods
-    val epbuf = new ListBuffer[ValDef]
+    val epbuf = ListBuffer[ValDef]()
```

Comment:
`ListBuffer[ValDef]` is unambiguous. Meaning there is no question whether it is a reference to an object or a constructor call here because of the type parameters. Can we drop `()` here in future?